### PR TITLE
Yeasmin

### PR DESCRIPTION
**Pull request by YeasminJaman**
First class create-yeasmin

### CODE REVIEW

File: src/GroupStudy/Yeasmin.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package GroupStudy;
+
+public class Yeasmin {
+    int id=100;
+
+    public static void main(String[] args) {
+        Yeasmin y=new Yeasmin();
```
Comment (MobassaraTabassum) [inline src/GroupStudy/Yeasmin.java:7]: please use a valid object name

**Review by MobassaraTabassum — CHANGES_REQUESTED** [review]
check it

**Comment by YeasminJaman** [comment]
I fix it

**Comment by YeasminJaman** [comment]
done

**Comment by YeasminJaman** [comment]
I am done

**Comment by YeasminJaman** [comment]
I fix it yeasmin

**Pull request merged**